pkgs/development/compilers/chicken/5/overrides.nix

```diff
@@ -14,9 +14,12 @@ let
     (addPkgConfig old) // (addToBuildInputs pkg old);
   addToPropagatedBuildInputsWithPkgConfig = pkg: old:
     (addPkgConfig old) // (addToPropagatedBuildInputs pkg old);
+  broken = old: { meta = old.meta // { broken = true; }; };
   brokenOnDarwin = old: { meta = old.meta // { broken = stdenv.isDarwin; }; };
 in {
-  allegro = addToBuildInputsWithPkgConfig [ pkgs.allegro5 pkgs.libglvnd ];
+  allegro = old:
+    (brokenOnDarwin old)
```

Comment:
Next time your mark a package broken can you add a comment indicating the failure so if someone fix an error it can see if this error appear in other place ?

Reply:
I cannot do that for all 100~ packages . You can see more failures here: https://hydra.nixos.org/eval/1793400?filter=chickenPackages_5.chickenEggs.&compare=1793383&full=#tabs-still-fail

Since I dont have access to a darwin machine, your work in fixing those packages is appreciated.

Reply:
> Since I dont have access to a darwin machine, your work in fixing those packages is appreciated.

https://github.com/winterqt/darwin-build-box.

Np placing this kind of link can be good also.